### PR TITLE
some pytest on math folder

### PR DESCRIPTION
**Pull request by moita69**
Hera are some pytest to simple math algorithms.

### CODE REVIEW

File: maths/3n+1.py [hunk 1]
```diff
@@ -32,6 +32,14 @@ def main():
         )
     )
 
+def testn31():
```
Comment (cclauss) [inline maths/3n+1.py]: `test_n31()` please.  Also, please remove blank lines 37 and 39.

File: maths/abs.py [hunk 2]
```diff
@@ -18,5 +18,13 @@ def main():
     print(abs_val(-34))  # = 34
 
 
+def testAbs():
```
Comment (cclauss) [inline maths/abs.py]: `test_abs_val()` please.  Also, please remove blank lines 23 and 25.

File: maths/average_mean.py [hunk 3]
```diff
@@ -12,5 +12,13 @@ def main():
     print(average([2, 4, 6, 8, 20, 50, 70]))
 
 
+def testAverage():
```
Comment (cclauss) [inline maths/average_mean.py]: `test_average()` please.  Also, please remove blank lines 17 and 19.

File: maths/average_mean.py [hunk 4]
```diff
@@ -12,5 +12,13 @@ def main():
...
+    assert 12.0 == average([3, 6, 9, 12, 15, 18, 21])
+
+    assert 20 == average([5, 10, 15, 20, 25, 30, 35])
+
+    assert 4.5 == average([1, 2, 3, 4, 5, 6, 7, 8])
+
 if __name__ == "__main__":
     main()
+    testAverage()
```
Comment (cclauss) [inline maths/average_mean.py]: Leave just one blank line at the end of Python files.

**Comment by cozek** [comment]
Thank you for your contribution. However, I think the standard for tests in this repo is doctests.
I think you should add doctests instead.
Do read CONTRIBUTION.md for more clarification.

**Comment by cclauss** [comment]
@cozek We need a pytest expert to help us understand why our non-doctest tests are not being discovered.  https://docs.pytest.org/en/latest/goodpractices.html#test-discovery

**Pull request merged**